resources/web/map/js/trajectory.js

```diff
@@ -0,0 +1,132 @@
+import mapboxGlStyleSwitcher from 'https://cdn.skypack.dev/mapbox-gl-style-switcher';
+
+mapboxgl.accessToken = "pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw";
```

Comment:
Where does this token come from, do we need register our own token?

Reply:
yea this was a token i just made a free account for - should be good for 50k map loads (each time toolbox runs counts as an init / load, tile requests dont count),

not sure best way to pass the token for build... (reading from a file which is only created in private CI ?)

Reply:
This could be a secret stored in swift-toolbox-ci -- we could dump the environment variable into a file before the CI runs.

Reply:
hmmm is there a way to avoid affecting local environment?
was avoiding that as setting up locally might include an extra step to fetch token 🤔

Reply:
I think the best option is to just not enable the map if there isn't a secret or api key available.  For local development we can store the key in AWS secrets manager and add some cargo-make scripts to fetch and populate the file locally.

Reply:
This is an interesting situation. In the past we had discussed adding some "paid" for features that could be enabled using a serial number or something similar but needing to inject secrets into the app also seems like something we will encounter more than once. So I could see it worthwhile investing in a scalable solution. I think we have inadvertently borked this key since this branch is public and I think dumping to plaintext also runs a similar risk. Maybe we could inject keys in the swift-toolbox-ci repo and encrypt them somehow, chatgpt lead me down a rabbit hole which lead me to this cipher crate: https://github.com/RustCrypto/AEADs/tree/master/chacha20poly1305

Reply:
I think as long as the key is not easy to dig out of the compiled binary (either obfuscated in a rust string constant) or in the QML resources it should be sufficiently protected for our purposes. See https://docs.mapbox.com/help/troubleshooting/private-access-token-android-and-ios/ -- we probably should rotate our keys occasionally, this will break older releases, but it'd be the lowest friction method of making sure the key doesn't leak (and, we should probably support injecting a custom API key for older builds).  We could also potentially tie a MapBox API key to a Skylark account, but that's a whole 'nother level of complexity.